README.md

```diff
@@ -833,6 +833,26 @@ when you need to retrieve a single record by some attributes.
   Mitigate code duplication by using partial templates and layouts.
 <sup>[[link](#partials)]</sup>
 
+* <a name="no-instance-variables-in-partials"></a>
+  Avoid using instance variables in partials, pass a local variable to `render` instead. Partial may be used
```

Comment:
Can I suggest some small grammar changes:

**The** partial may be used in a different controller or action, where the variable can have a different name or **is absent**. In these cases **an** undefined instance variable will not raise an exception whereas **a** local variable will.

Reply:
Yes, this way it reads so much better, thanks!